test/sql/size_utils.sql

```diff
@@ -5,7 +5,13 @@
 \ir include/insert_two_partitions.sql
 
 SELECT * FROM hypertable_detailed_size('"public"."two_Partitions"');
-SELECT * FROM indexes_relation_size('"public"."two_Partitions"');
+SELECT * FROM hypertable_index_relation_size('"public"."two_Partitions_device_id_timeCustom_idx"');
```

Comment:
Nit: You don't need the quotes.
```suggestion
SELECT * FROM hypertable_index_relation_size('public.two_Partitions_device_id_timeCustom_idx');
```